test/onnx/test_op_consistancy.py

```diff
@@ -0,0 +1,460 @@
+# Owner(s): ["module: onnx"]
+
+"""Test consistency between torch.onnx exported operators and torch operators.
+
+Usage:
+
+    pytest test/onnx/test_op_consistancy.py
+
+    To run tests on a specific operator (e.g. torch.ceil):
+
+    pytest test/onnx/test_op_consistancy.py -k ceil
```

Comment:
Nice example. This comment can also point users to 

https://github.com/pytorch/pytorch/wiki/Running-and-writing-tests